tests/ert3/console/integration/test_cli.py

```diff
@@ -512,3 +513,25 @@ def test_cli_validation_stages_command(base_ensemble_dict, workspace, capsys):
     capture = capsys.readouterr()
     assert "Error while loading stages configuration data:" in capture.out
     assert "str type expected" in capture.out
+
+
+def test_no_service(tmpdir):
```

Comment:
Should we rename these tests to include `check` and add some tests for `start` as well?